src/Uno.UI/UI/Xaml/Data/ItemIndexRange/ItemIndexRangeHelper.h.mux.cs

```diff
@@ -0,0 +1,39 @@
+﻿// MUX Reference ItemIndexRangeHelper.h
+
+using System.Collections.Generic;
+
+namespace DirectUI.Components;
+
+internal static partial class ItemIndexRangeHelper
+{
+	internal enum IndexLocation
+	{
+		Before,
+		After,
+		Inside
+	}
+
+	internal struct Range //TODO:MZ:WinUI uses struct here
```

Comment:
Maybe add a link in this comment to the related item or issue for this TODO

Reply:
The todo does not apply anymore, as we are now using struct as well